### PR TITLE
Migrate OH T5-large training to torch.compile

### PR DESCRIPTION
**Pull request by chaojun-zhang**
# What does this PR do?

Add an example on 8 HPUs on Gaudi2 with DeepSpeed-ZeRO2 to fine-tune t5-large

Fixes # (issue)

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you make sure to update the documentation with your changes?
- [ ] Did you write any new necessary tests?

### CODE REVIEW

**Comment by astachowiczhabana** [comment]
@libinta  this commit is required with next OH release

**Comment by vidyasiv** [comment]
@chaojun-zhang , please paste output of modified commands here run w/ latest 1.19 docker.

File: examples/summarization/README.md [hunk 1]
```diff
@@ -209,6 +209,35 @@ PT_HPU_MAX_COMPOUND_OP_SIZE=512 python ../gaudi_spawn.py \
...
+      --save_strategy no \
+      --throughput_warmup_steps 15 \
+      --model_name_or_path t5-large \
+      --source_prefix '"summarize:"' \
+      --dataset_name cnn_dailymail \
+      --dataset_config '"3.0.0"' \
+      --output_dir /tmp/tst-summarization \
+      --per_device_train_batch_size 4 \
+      --per_device_eval_batch_size 4 \
+      --max_train_samples 2000  \
```
Comment (vidyasiv) [inline examples/summarization/README.md:236]: @chaojun-zhang  can you also ensure the command line options provide optimal performance for torch compile mode with this model?

**Review by regisss — APPROVED** [review]
LGTM. I'll merge once @vidyasiv's comments are addressed.

**Comment by regisss** [comment]
@chaojun-zhang Do you get similar performance with torch.compile as with lazy mode?

**Comment by chaojun-zhang** [comment]
> @chaojun-zhang Do you get similar performance with torch.compile as with lazy mode?

Yes, but I can't apply the hpu machine these two days

**Comment by chaojun-zhang** [comment]
> @chaojun-zhang Do you get similar performance with torch.compile as with lazy mode?

 1.19 latest docker + torch.compile
***** train metrics *****
  epoch                       =        3.0
  max_memory_allocated (GB)   =      92.32
  memory_allocated (GB)       =       4.28
  total_flos                  = 25164140GF
  total_memory_available (GB) =      94.62
  train_loss                  =     3.6603
  train_runtime               = 0:01:49.68
  train_samples               =       2000
  train_samples_per_second    =    222.287
  train_steps_per_second      =      1.482

***** eval metrics *****
  epoch                       =        3.0
  eval_gen_len                =      127.0
  eval_loss                   =      2.406
  eval_rouge1                 =     38.157
  eval_rouge2                 =    16.1902
  eval_rougeL                 =    25.6348
  eval_rougeLsum              =    35.7379
  eval_runtime                = 0:26:25.96
  eval_samples                =      13368
  eval_samples_per_second     =      8.513
  eval_steps_per_second       =      0.054
  max_memory_allocated (GB)   =      92.32
  memory_allocated (GB)       =       4.28
  total_memory_available (GB) =      94.62

**Pull request merged**